src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -509,6 +509,12 @@ var ReactCompositeComponentMixin = {
       return;
```

Comment:
I guess we actually already fail silently here (unset in the same place as ._instance). So we don't need this check.